### ISSUE TITLE
Project abandoned?

### ISSUE THREAD

**Issue by neverfox** [juxt/aero#61]
I'm wondering if aero has been abandoned. I certainly hope not and that everything is okay at JUXT, but there are three PRs that have been sitting around without comment for a while and no real activity. They're all failing but that seems to be entirely due to an issue with rake in the build process. An status update would be appreciated, as I don't think it's an overstatement to say that this is probably the top config library for Clojure right now.

**Comment by malcolmsparks** [comment]
It definitely hasn't been abandoned. It is a key library in all our projects here at JUXT. Fear not, your issue has woken up the library fairies here and they are busily dusting shelves and cleaning dishes.

**Comment by neverfox** [comment]
```
> $ lein ancient                                                                                                                                                                                   
[aero "1.1.3"] is available but we use "1.1.2"
```
Aww yeah!
